gradle/dependencies.gradle

```
@@ -50,7 +50,7 @@ ext {
       kotlinStdLib              : '1.3.72',
       ankoCommon                : '0.10.0',
       firebaseCore              : '16.0.7',
-      crashlytics               : '2.9.9',
+      crashlytics               : '17.1.0',
```

should it be renamed according firebase names, just like with `firebaseCore `
```suggestion
      firebaseCrashlytics       : '17.1.0',
```

btw https://firebase.google.com/support/release-notes/android#kotlin+ktx
> The Firebase Android library firebase-core is no longer needed. This SDK included the Firebase SDK for Google Analytics.

---

This is crashlytics, not core.

---

Ah, I think I got it, removed the dep :D